iot-hub/Quickstarts/InvokeDeviceMethod/Program.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 // This application uses the Azure IoT Hub service SDK for .NET
-// For samples see: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/iothub/service
+// For samples see: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/service
```

Comment:
I updated these as well so when we update the c# repo we won't have to update these again.